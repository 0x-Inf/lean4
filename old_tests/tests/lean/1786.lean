theorem nil_subset : (true ∧ true) = true := by simp
open list

example (x : ℕ) : x = x := by simp [nil_subset]
