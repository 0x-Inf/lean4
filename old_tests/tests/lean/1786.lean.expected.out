1786.lean:4:30: error: ambiguous overload, possible interpretations
    nil_subset
    list.nil_subset
state:
x : ℕ
⊢ x = x
